src/script/lua_api/l_env.cpp

```diff
@@ -498,7 +498,7 @@ int ModApiEnvMod::l_dig_node(lua_State *L)
 	}
 	// Dig it out with a NULL digger (appears in Lua as a
 	// non-functional ObjectRef)
-	bool success = scriptIfaceNode->node_on_dig(pos, n, NULL);
+	bool success = scriptIfaceNode->node_on_dig(pos, n, NULL, PointedThing());
```

Comment:
Yes, I pass a ``POINTEDTHING_NOTHING`` because node_on_dig expects a PointedThing but this function (l_dig_node) is generally called when it wasn't a player that dug the node, as far as I understand it.